frontend/src/views/SQR/Form.vue

```diff
@@ -225,11 +233,13 @@ export default class Form extends Vue {
     }
   ];
   cancel: number | null = null;
-  cancelComments = {};
+  cancelComments: { [key: number]: string } = {};
```

Comment:
I'm not sure we need to keep both comments separate. Why not to use the same property for both cases?